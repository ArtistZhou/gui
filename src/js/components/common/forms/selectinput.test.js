import React from 'react';
import renderer from 'react-test-renderer';
import SelectInput from './selectinput';

describe('SelectInput Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SelectInput attachToForm={() => {}} menuItems={[]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
