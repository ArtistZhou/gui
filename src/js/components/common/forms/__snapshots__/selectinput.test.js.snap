// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiFormControl-root-1"
>
  <label
    className="MuiFormLabel-root-16 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13"
    data-shrink={false}
    htmlFor="simple-select"
  />
  <div
    className="MuiInputBase-root-43 MuiInput-root-30 MuiInput-underline-34 MuiInputBase-formControl-44 MuiInput-formControl-31"
    onClick={[Function]}
  >
    <div
      className="MuiSelect-root-23"
    >
      <div
        aria-haspopup="true"
        aria-pressed="false"
        className="MuiSelect-select-24 MuiSelect-selectMenu-27 MuiInputBase-input-53 MuiInput-input-38"
        id="select-selector"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          dangerouslySetInnerHTML={
            Object {
              "__html": "&#8203;",
            }
          }
        />
      </div>
      <input
        id="simple-select"
        name="selector"
        type="hidden"
      />
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-60 MuiSelect-icon-29"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M7 10l5 5 5-5z"
        />
      </svg>
    </div>
  </div>
</div>
`;
